docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -165,3 +164,22 @@ list_adapters_component_wise = pipe.get_list_adapters()
 list_adapters_component_wise
 {"text_encoder": ["toy", "pixel"], "unet": ["toy", "pixel"], "text_encoder_2": ["toy", "pixel"]}
 ```
+
+## Fusing multiple adapters into the model
```

Comment:
I think it may flow better if this section was under the "Combine multiple adapters" section since the two are kind of related.

Reply:
I think that's different, because in theory one can fuse just one adapter also. So maybe the title can be changed of the whole session to just "Fusing adapters into the model"

Reply:
That sounds good!